server/src/main/resources/db/changelog/20171017062314-add-target-index-on-job-table.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+    <changeSet id="20171017062314-1" author="vrjain">
+        <preConditions onFail="MARK_RAN">
```

Comment:
Why do we need to have a precondition on this?  Are we worried that customers/hosted have already applied this index?

Reply:
IT has already applied this index to hosted. They had major issues and added it to address it.

Reply:
@mstead apologies, I assumed the card had the appropriate name, so I went with that. checked with tdarby, the name matches with this PR.